lib/plugins/aws/deployList/tests/index.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const sinon = require('sinon');
+const BbPromise = require('bluebird');
+const expect = require('chai').expect;
+const AwsDeployList = require('../index');
+const AwsProvider = require('../../provider/awsProvider');
+const Serverless = require('../../../../Serverless');
+
+describe('listDeployments', () => {
```

Comment:
We've used the name of the plugin here --> `AwsDeployList` in other plugin related tests (https://github.com/serverless/serverless/blob/master/lib/plugins/aws/deployFunction/tests/index.js#L14). Not sure if this is best practice though.